### ISSUE TITLE
posterior-from-I-model.py

### ISSUE THREAD

**Issue by gyq-123** [birc-aeh/coalhmm#10]
I encountered an error when running the script "posterior-from-I-model.py", I looked up the example file and there was no error. Could it be caused by the previous problem in the generation of ziphmm file.
[[2.48913145e-01 3.62285026e-04 1.00000000e+00]
 [2.48512203e-01 4.95932455e-04 1.00000000e+00]
 [2.48061925e-01 6.46024893e-04 1.00000000e+00]
 [2.47548244e-01 8.17252102e-04 1.00000000e+00]
 [2.46950001e-01 1.01666637e-03 1.00000000e+00]
 [2.46233174e-01 1.25560867e-03 1.00000000e+00]
 [2.45337600e-01 1.55413334e-03 1.00000000e+00]
 [2.44140092e-01 1.95330273e-03 1.00000000e+00]
 [2.42311393e-01 2.56286901e-03 1.00000000e+00]
 [2.37554948e-01 4.14835053e-03 1.00000000e+00]]
Traceback (most recent call last):
  File "/home/ug1231/software/coalhmm-master/scripts/posterior-from-I-model.py",                                                                              line 164, in <module>
    main()
  File "/home/ug1231/software/coalhmm-master/scripts/posterior-from-I-model.py",                                                                              line 133, in main
    mpi, mT, mE, time_breaks = get_model_matrices(model, coal_rate, rec_rate, sp                                                                             lit_time)
  File "/home/ug1231/software/coalhmm-master/scripts/posterior-from-I-model.py",                                                                              line 49, in get_model_matrices
    pi, T, E = model.run(r, c, time_breakpoints, M, col_map=FIXED_COL_MAP)
  File "/home/ug1231/.local/lib/python2.7/site-packages/coalhmm/model.py", line                                                                              339, in run
    assert all(allclose(row.sum(), 2.0) for row in Em)
AssertionError

**Comment by mailund** [comment]
Yeah, that is a consistency check that is failing. I am not sure what is going on. Could you try using [Jocx](https://github.com/jade-cheng/Jocx) instead? It is more recent and I have a bit more confidence in that.
